pkg/pipeline/ChartService.go

```diff
@@ -47,6 +48,39 @@ import (
 	"time"
 )
 
+type NameIncludesExcludes struct {
+	Names []string `json:"names"`
+}
+type Specs struct {
+	PatchJson string `json:"patchJson"`
+}
+type Tasks struct {
+	Spec Specs `json:"spec"`
+}
+type BulkUpdatePayload struct {
+	Includes           NameIncludesExcludes `json:"includes"`
+	Excludes           NameIncludesExcludes `json:"excludes"`
+	EnvIds             []int                `json:"envIds"`
+	Global             bool                 `json:"global"`
+	DeploymentTemplate Tasks                `json:"deploymentTemplate"`
+}
+type BulkUpdateScript struct {
+	ApiVersion string            `json:"apiVersion" validate:"required"`
+	Kind       string            `json:"kind" validate:"required"`
+	Payload    BulkUpdatePayload `json:"payload" validate:"required"`
+}
+type BulkUpdateRequest struct {
+	Task   string           `json:"task"`
```

Comment:
what is this task?

Reply:
this is used for the "deploymentTemplate" part in input script(refer attached image) and I named it Task for giving it more generic touch...since we have plans for supporting more tasks with Bulk Update in future
<img width="775" alt="Screenshot 2021-07-08 at 12 56 33 AM" src="https://user-images.githubusercontent.com/84493919/124817799-df9b2d80-df87-11eb-9fe4-e3724fabcabb.png">
. Should I change this?